sources/packages/backend/workflow/test/test-utils/execute-workflow-instance-utils.ts

```diff
@@ -27,3 +27,25 @@ export async function executeFulltimeAssessmentForProgramYear(
     requestTimeout: PROCESS_INSTANCE_CREATE_TIMEOUT,
   });
 }
+
+/**
+ * Executes the parttime-assessment-* BPMN workflow.
+ * @param programYear program year to be invoked.
+ * @param assessmentConsolidatedData assessment data.
+ * @returns result of the workflow execution.
+ */
+export async function executePartTimeAssessmentForProgramYear(
+  programYear: string,
+  assessmentConsolidatedData: AssessmentConsolidatedData,
+): Promise<CreateProcessInstanceWithResultResponse<CalculatedAssessmentModel>> {
+  return ZeebeMockedClient.getMockedZeebeInstance().createProcessInstanceWithResult<
+    AssessmentConsolidatedData,
+    CalculatedAssessmentModel
+  >({
+    bpmnProcessId: `parttime-assessment-${programYear}`,
+    variables: {
+      ...assessmentConsolidatedData,
+    },
+    requestTimeout: PROCESS_INSTANCE_CREATE_TIMEOUT,
+  });
+}
```

Comment:
Is the `executePartTimeAssessmentForProgramYear` same as `executeFulltimeAssessmentForProgramYear` changing only the `parttime/fulltime` assessment? If yes we can have a centrailized not exported method to keep the logic and the exported methods as they are right now just providing the `bpmnProcessId` as a parameter.

Reply:
I am not sure if the idea was achieved. With the change now we have `fulltime-assessment-${PROGRAM_YEAR}` or `parttime-assessment-${PROGRAM_YEAR}` repeated everywhere and it can be centralized. My idea was option A or B below.

Option A:
Keep both methods `executePartTimeAssessmentForProgramYear` and `executeFulltimeAssessmentForProgramYear` calling a "not exported method" with the logic.

Option B:
Create a single exported method (e.g. `executeAssessment`) that would receive as parameter "Part-time/Full-time".